language-plutus-core/test/Normalization/Type.hs

```diff
@@ -42,4 +58,5 @@ test_typeNormalization =
     testGroup "typeNormalization"
         [ testCase     "appAppLamLam"               test_appAppLamLam
         , testProperty "normalizeTypesInIdempotent" test_normalizeTypesInIdempotent
+        , testCase     "bug"                        test_catamorphism
```

Comment:
Please call the test "catamorphism" rather than "bug".

Reply:
I suggest naming regression test cases after the issue reporting the bug.

Reply:
> I suggest naming regression test cases after the issue reporting the bug.

This is better indeed.